### PR TITLE
Line numbers overlapping over code #419 

### PR DESCRIPTION
**Pull request by adamovivan**

### CODE REVIEW

File: web/src/main/frontend/src/components/ReviewCodeChallengeQuestion.js [hunk 1]
```diff
@@ -10,6 +10,17 @@ import 'codemirror/mode/jsx/jsx';
 import 'codemirror/mode/php/php';
 
 class ReviewCodeChallengeQuestion extends Component {
+  componentDidMount() {
+    setTimeout(
```
Comment (aberonja) [inline web/src/main/frontend/src/components/ReviewCodeChallengeQuestion.js:14]: setTimeout? 🤔

Comment (adamovivan) [reply web/src/main/frontend/src/components/ReviewCodeChallengeQuestion.js:14]: There is some problem with the CodeMirror component. I tried I lot of things and nothing worked. Some people on the internet solved the problem in that way.

Comment (aberonja) [reply web/src/main/frontend/src/components/ReviewCodeChallengeQuestion.js:14]: Please, leave a comment then why you did it like this.

Comment (adamovivan) [reply web/src/main/frontend/src/components/ReviewCodeChallengeQuestion.js:14]: I leave some time for everything to initialize completely, so then trigger change state. After the state is changed CodeMirror bug disappears.

**Pull request merged**